provelo_custom_invoice_auto_open/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Coop IT Easy SC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    group_account_invoice_auto_open = fields.Boolean(
+        string="Automatically Open Invoices",
+        implied_group="provelo_custom_invoice_auto_open"
+        ".account_invoice_auto_open_group",
+    )
```

Comment:
as i understand, either this setting is true and the invoices open automatically for all employees, or it is false, and invoices open automatically only for members of the `provelo_custom_invoice_auto_open.account_invoice_auto_open_group` group. is this correct?

i think this should be explained in the readme. (and even maybe add a test for this?)

Reply:
It is tested in 

- `test_create_invoice_opens_it_for_auto_invoice_group`
- `test_create_invoice_does_not_open_it_without_auto_invoice_group`

But I can add a comment.